doc/use_cli_in_airgapped_clouds.md

```diff
@@ -4,13 +4,13 @@
 While Azure CLI is not fully supported in an airgapped cloud which has no public internet connectivity, most commands can run without issues if the underlying service is supported in that cloud.
 
 Here is a list of known CLI features that are not supported in airgapped clouds:
-* Manage extensions with extension names. For instance, you can not install the `azure-devops` extension with `az extension add --name azure-devops`. You need to provide a local path to a wheel file or a URL for downloading the wheel file that is accessible in the airgapped cloud with `--source` option.
+* Manage extensions with extension names. For instance, you cannot install the `azure-devops` extension with `az extension add --name azure-devops`. You need to provide a local path to a wheel file or a URL for downloading the wheel file that is accessible in the airgapped cloud with `--source` option.
```

Comment:
What's the difference between "can not" and "cannot"?

Reply:
> Both cannot and can not are acceptable spellings, but the first is much more usual. You would use can not when the "not" forms part of another construction such as "not only." For example:
> 
> These green industries can not only create more jobs, but also promote sustainable development of the land.

Link: https://www.lexico.com/grammar/cannot-or-can-not